tests/test_octodns_provider_googlecloud.py

```diff
@@ -280,6 +284,9 @@ def __next__(self):
 
 
 class TestGoogleCloudProvider(TestCase):
+    def test_trailing_dot(self):
+        self.assertEqual(add_trailing_dot('unit.tests'), 'unit.tests.')
```

Comment:
Best to explicitly test both cases even though one of them was otherwise being hit by tests elsewhere.

```suggestion
        self.assertEqual(add_trailing_dot('unit.tests'), 'unit.tests.')
        self.assertEqual(add_trailing_dot('unit.tests.'), 'unit.tests.')
```